src/components/DesignTheme/View/Commons/Venue.jsx

```diff
@@ -0,0 +1,47 @@
+import { useDispatch, useSelector } from 'react-redux';
+import React from 'react';
+import { getContent } from '@plone/volto/actions';
+import { flattenToAppURL } from '@plone/volto/helpers';
+
+/**
+ * OfficeCard view component class.
```

Comment:
Occhio ai copia-incolla. Ma poi sti commenti quando sono così vuoti possiamo pure pensare di toglierli volendo, ci sono perché in Volto all'inizio li mettevano ovunque, non aggiungono nulla a quello che ti dice la firma della funziona, hanno smesso anche nel core.